CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ Current
     * Add a configurable property named "druid_uncovered_interval_limit"
     * Add new response error messages as needed by Partial Data V2
 
+- [Merge Druid Response Header into Druid Response Body Json Node in AsyncDruidWebServiceImplV2](https://github.com/yahoo/fili/pull/267)
```

Comment:
Should also call out the change to the `AsyncDruidWebServiceClientImpl`, in that it now takes a strategy for building the JSON from the entire response. (since that's what the ability we're adding hinges on)